styles/main.css

```diff
@@ -337,13 +344,13 @@ a .paperclip {
     position: relative;
 }
 
-[title|="Back to top"] {
+a[href="#top"] {
```

Comment:
I'm actually not sure why the selector you had doesn't work!  But here's an alternative that does...